lists/global.csv

```diff
@@ -835,7 +835,7 @@ http://www.oic-oci.org/,REL,Religion,2014-04-15,citizenlab,Updated by OONI on 20
 http://www.oicc.org/,REL,Religion,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
 http://www.oikoumene.org/,REL,Religion,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
 https://www.okcupid.com/,DATE,Online Dating,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
-https://www.om.org/,REL,Religion,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
+https://www.omusa.org/,REL,Religion,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
```

Comment:
I still see content on https://www.om.org/. I think omusa.org is the US chapter of OM similarly to how there is a Canadian chapter on omcanada.org, see: https://twitter.com/OM_Canada.

I suggest you add this site to the US testing list or to the country lists.